frontend/src/metabase/components/AppBanner.jsx

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable react/prop-types */
+import { connect } from "react-redux";
+import React from "react";
+import { t } from "ttag";
+import Banner from "metabase/components/Banner";
+import { getUserIsAdmin } from "metabase/selectors/user";
+import { useLicense } from "metabase/admin/settings/hooks/use-license";
+
+const PAST_DUE_ERROR = t`⚠️ We couldn't process payment for your account. Please [review your payment settings](https://store.metabase.com/) to avoid service interruptions.`;
```

Comment:
Super minor, but I worry about leaving the URL as part of the string which will be localized by the community, because it deals with paid services, etc. I wonder if it isn't worth interpolating its value, not making the URL translatable (I have `sprintf` mental model for `t`).

Reply:
Good catch, I've now fixed it here:
https://github.com/metabase/metabase/blob/24bdd4fb3a124b358e229a026817183f8b493190/frontend/src/metabase/components/AppBanner.jsx#L11-L28